middleware/lib/claim-client.js

```diff
@@ -144,7 +144,12 @@ class ClaimClient {
    *
    * @param {string} vin - The VIN of the claim to delete.
    */
-  async deleteClaim(vin) {
+  async deleteClaim(vin, { vehicle = {} }) {
```

Comment:
@ej3076 - If @dsifford agrees we should have this check for delete claim, this will require an update to send the VIN in the body so it can check if it exists/matches the VIN parameter, similar to editing a claim.

Reply:
I'm not sure I understand what you're describing here.

Reply:
You can disregard this.  This is not needed - I was trying to mimic the edit claim errors too closely.